### DISCUSSION TITLE
Poles Figures MTEX Erreur

### DISCUSSION THREAD

**Discussion by Mabroukimedcea** [mtex-toolbox/mtex#1750] [Ask Anything]
Hello, 

I 'am trying to plot poles figures for my EBSD data with MTEX,
I am trying that by the code below 
%% Plot poles Figures 
% the selected crystal directions
h = Miller({1,0,0},{1,1,0},{1,1,1},MB_EBSD.CS);
% plot their positions with respect to specimen coordinates
plotPDF(MB_EBSD.orientations,h, 'antipodal', 'contourf', 0:1:10.0, 'minmax', 'colorRange',[0,10]);
mtexColorbar

I get this erreur, could anyone help me to bypass it please, thanks in advanced 

@ralfHielscher 
MTEX
Poles figures 
-------------------------------------------------------------------------------------------------------
Invalid MEX-file '/home/catC/mm264941/mtex-5.8.2/extern/nfft_openMP/mtex-5.7.0/nfsftmex.mexa64': libmvec.so.1: cannot open shared object file: No such file or directory

Error in S2FunHarmonic.quadrature (line 72)
  nfsftmex('precompute', bw, 1000, 1, 0);

Error in vector3d/calcDensity (line 36)
sF = 4*pi * S2FunHarmonic.quadrature(v,ones(size(v)),varargin{:});

Error in vector3d/smooth (line 39)
    cdata = calcDensity(v(:),S2G,'halfwidth',5*degree,varargin{:});

Error in vector3d/contourf (line 31)
  [varargout{1:nargout}] = v.smooth(data,'contours',10,'LineStyle','-','LineColor','k',varargin{:});

Error in vector3d/plot (line 111)
    [varargout{1:nargout}] = v.contourf(varargin{:});

Error in vector3d/subsref (line 20)
    [varargout{1:nargout}] = builtin('subsref',v,s);

Error in orientation/plotPDF (line 138)
  [~,cax] = r.plot(repmat(data,[1 numSym(ori.SS)*length(sh) 1]),...
-------------------------------------------------------------------------------------------------------------------------------

**Comment by kilir** [comment]
Hi,
probably your glibc is too old.

See here:
https://github.com/mtex-toolbox/mtex/issues/402

Cheers,
Rüdiger

**Reply by Mabroukimedcea** [reply]
Hi Rüdiger, 
thank you so much for your answer 
howerver is there any simple solution to bypass such a problem
thanks again

**Reply by kilir** [reply]
Yes,
see here:https://github.com/mtex-toolbox/mtex/issues/402#issuecomment-462822048

Hope this helps,
Rüdiger

**Reply by Mabroukimedcea** [reply]
Thanks again Rüdiger and last question please ,
I saw your answer and one more question, please, as i can not change my glibc, could you tell me please how to compile the nfft without vectorization, thank you again

Mohamed
See here: https://groups.google.com/d/msg/mtexmail/0LkUv2XiPJI/76N4CydxAwAJ
If you can't change to a newer glibc, you might have to compile nfft for this setup without vectorization/openMP support, which will most likely be slower in some tasks.
